src/types/fullscreen_shell_v1.zig

```diff
@@ -0,0 +1,32 @@
+const wlr = @import("../wlroots.zig");
+
+const wayland = @import("wayland");
+const wl = wayland.server.wl;
+const zwp = wayland.server.zwp;
+
+pub const FullscreenShellV1 = extern struct {
+    pub const event = struct {
+        pub const PresentSurface = extern struct {
+            client: *wl.Client,
+            surface: ?*wlr.Surface,
+            method: zwp.FullscreenShellV1.PresentMethod,
```

Comment:
Hmm I can't find a solution how to fix this, not sure if I need to add the enum in the bindings, if it's a problem with how the test works or just an error in the name